docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -42,6 +34,20 @@ A node running YugabyteDB Anywhere is expected to meet the following requirement
 - 100 GB SSD disk or more
 - 64-bit CPU architecture
 
+### Kubernetes-based installations
+
+A Kubernetes node is expected to meet the following requirements: 
+
+- 5 cores (minimum) or 8 cores (recommended)
+
+- 15 GB RAM (minimum)
+
+- 250 GB SSD disk (minimum) allocated to YugabyteDB Anywhere 
```

Comment:
"allocated to YugabyteDB Anywhere" sounds technically incorrect here because we are talking about the underlying kubernetes node.

Reply:
Deleted "allocated to YugabyteDB Anywhere"